man/aptly.1

```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "APTLY" "1" "March 2017" "" ""
+.TH "APTLY" "1" "2017-03-23" "" ""
```

Comment:
Took a while to get ronn up and running. Apparently using mustache in version 1.0.4 caused errors, so I had to revert that to 1.0.3. Nevertheless, it now seems to work and I've fixed the man page in 4ddf85b.